aframe/src/new-location-based/gps-new-camera.js

```diff
@@ -143,9 +143,9 @@ AFRAME.registerComponent("gps-new-camera", {
   _displayError: function (error) {
     const arjs = this.el.sceneEl.systems["arjs"];
     if (arjs) {
-      arjs._displayErrorPopup(msg);
```

Comment:
this make sense!

Reply:
this fix error reported in [comment](https://github.com/AR-js-org/AR.js/pull/507#issuecomment-1430150668)

Reply:
@nickw1 this is fixed i tested again and the error is not shown anymore.

Reply:
@kalwalt ok great, thanks.

Reply:
with Typescript the compiler will alert you, and as a consequence this kind of bugs are solved immediately.